__tests__/app.test.js

```diff
@@ -456,3 +456,57 @@ describe("DELETE /api/comments/:comment_id", () => {
       });
   });
 });
+
+describe("GET /api/users", () => {
+  test("responds with status code 200", () => {
+    return request(app).get("/api/users").expect(200);
+  });
+  test("responds with an array of user objects with username, name and avatar_url properties", () => {
+    return request(app)
+      .get("/api/users")
+      .then((response) => {
+        const { users } = response.body;
+        expect(users.every((user) => user.hasOwnProperty("username"))).toBe(
+          true
+        );
+        expect(users.every((user) => user.hasOwnProperty("name"))).toBe(true);
+        expect(users.every((user) => user.hasOwnProperty("avatar_url"))).toBe(
+          true
+        );
+      });
+  });
+  test("responds with a correct array of user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .then((response) => {
+        const { users } = response.body;
+        const expected = [
+          {
+            username: "butter_bridge",
+            name: "jonny",
+            avatar_url:
+              "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg",
+          },
+          {
+            username: "icellusedkars",
+            name: "sam",
+            avatar_url:
+              "https://avatars2.githubusercontent.com/u/24604688?s=460&v=4",
+          },
+          {
+            username: "rogersop",
+            name: "paul",
+            avatar_url:
+              "https://avatars2.githubusercontent.com/u/24394918?s=400&v=4",
+          },
+          {
+            username: "lurker",
+            name: "do_nothing",
+            avatar_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          },
+        ];
+        expect(users).toEqual(expected);
```

Comment:
Extremely brittle test here. I would avoid doing a toEqual match like this. If the test data morphs in any way you will have to comb through this test. This means if the users have any more properties on them and if another user is added. At least if you're using a length check and doing a more generic toMatchObject check you have less to amend if the data changes.